include/CMakeLists.txt

```diff
@@ -10,9 +10,21 @@ target_include_directories(
             $<BUILD_INTERFACE:${OE_INCDIR}>
             $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/${CMAKE_INSTALL_INCLUDEDIR}>)
 install(
-  DIRECTORY openenclave/bits
-  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/
+  FILES openenclave/bits/attestation.h
+        openenclave/bits/defs.h
+        openenclave/bits/report.h
+        openenclave/bits/result.h
+        openenclave/bits/types.h
+  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/bits/
   COMPONENT OEHOSTVERIFY)
+install(DIRECTORY openenclave/bits
+        DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/)
+install(DIRECTORY openenclave/bits/edl
```

Comment:
Don't we install these sub directories when installing `openenclave/bits`?

Reply:
Yes, you're right. I have undone the change.